bot/seasons/easter/egg_hunt/constants.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+from discord import Colour
+
+from bot.constants import Channels, Client, bot
+
+
+GUILD = bot.get_guild(Client.guild)
+
+
+class EggHuntSettings:
+    start_time = int(os.environ["HUNT_START"])
+    end_time = start_time + 172800
```

Comment:
What does this integer represent? Could maybe put a comment here saying what it is.

Reply:
+48 hrs. Will add an inline comment, thanks.